Dependencies/MelonStartScreen/UnityEngine/CoreModule/ImageConversion.cs

```diff
@@ -2,6 +2,7 @@
 using System;
 using System.Runtime.InteropServices;
 using UnhollowerMini;
+using static MelonLoader.MelonStartScreen.Core;
```

Comment:
Can we not do this, just use Core.Logger further down

Reply:
Fine...

Reply:
https://github.com/LavaGang/MelonLoader/pull/197/commits/cb08a9238cbf255958dfb97e9317170df708569b